src/focus-trap-react.js

```diff
@@ -48,17 +53,82 @@ class FocusTrap extends React.Component {
     this.updatePreviousElement();
   }
 
+  // TODO: Need more test coverage for this function
+  getNodeForOption(optionName) {
+    const config = {
+      returnFocusOnDeactivate: true,
+      escapeDeactivates: true,
+      delayInitialFocus: true,
+      ...this.tailoredFocusTrapOptions,
+    };
+
+    const optionValue = config[optionName];
```

Comment:
Actually if you can replace `config` here with `this.tailoredFocusTrapOptions` then it will unblock the PR.

Reply:
Done!